test/modules/aws-ecr-integration.ts

```diff
@@ -203,7 +203,14 @@ describe('ECR Integration Testing', () => {
     `,
         (res: any[]) => {
           expect(res.length).toBe(3);
+          expect(res.map(i => i['size_in_mb']).reduce((a, b) => a + b, 0)).toBe(0);
```

Comment:
either we are pushing something wrong to the repo or this check is not working properly, the size should not be 0, I think. If you see [here](https://gallery.ecr.aws/docker/library/hello-world) the image tags tab it says 122MB

Reply:
size is 14kB in the link you provided though I am seeing 4kB, but both of those round to 0 in MB. I think it is okay as-is or should do you want to track kB or bytes? That will be too small when the image is in GB

Reply:
ohh I see, then that's ok, let's keep it as is!